cockpit_api/src/main/java/com/cockpit/api/controller/TeamController.java

```diff
@@ -140,11 +168,12 @@ public ResponseEntity deleteTeamMember(@PathVariable("id") Long id,
             return ResponseEntity.ok(HttpStatus.UNAUTHORIZED);
         }
     }
+
     // UNASSIGN a Team member
     @DeleteMapping(value = "/api/v1/team/{id}/unassignTeamMember/{teamMemberId}")
     public ResponseEntity<Object> unassignTeamMember(@PathVariable("id") Long id,
-                                           @PathVariable("teamMemberId") Long teamMemberId,
-                                           @RequestHeader("Authorization") String authHeader) {
+                                                     @PathVariable("teamMemberId") Long teamMemberId,
```

Comment:
It comes to me that unassign is an action of "update", but we created it as a DeleteMapping, how about modifying it to @PutMapping(should modify the unit test, too) ?

Reply:
done